frontend/src/components/Title.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import "./css/Title.css"
+
+export default function Title() {
+    return (
+        <div className={"titleContainer"}>
+            <h1> Capstone Texas Hold'em App </h1>
```

Comment:
My 2 cents: Whether Capstone or not, its Texas Hold'em, isn't it? Also the app...why not game or better only "Texas Hold'em"?
```suggestion
            <h1>Texas Hold'em</h1>
```